F21/rjuliang/LowerCaser/tests.tsv

```diff
@@ -15,4 +16,4 @@ KASIM	tr	kasım
 KASIM	en	kasim
 ΠΌΛΗΣ	el	πόλης
 官话	zh-Hans	官话
-ภาษาไทย	th	ภาษาไทย
+ภาษาไทย	th	ภาษาไทย
```

Comment:
I think your code is too clear for anybody with basic level of JavaSript skills can understand.
And at the the same time there are too many functions used, but overall it is very clear and up to the point.